features/admin.extensions.v1/configs/attribute.ts

```diff
@@ -226,6 +226,7 @@ export const attributeConfig: AttributeConfig = {
             return true;
         }
     },
+    isMobileNumberVerificationByPrivilegedUsersEnabled: true,
```

Comment:
Is it intentional to set this to true in onprem? afaiu, privileged users exist in Asgardeo and we need to set this to `true` in Asgardeo extensions, not here. Please correct me if im wrong though.

Reply:
No, in here the configuration "UserClaimUpdate.MobileNumber.EnableVerificationByPrivilegedUser" refers to  admins. Although the name suggests that it allows admins to initiate verification, what it actually does is enable admins to verify the code on behalf of users. Due to the ambiguity in its naming and the lack of UI support, we decided not to move this config to Asgardeo. However, since this capability existed in IS 7.0, we are retaining it in IS 7.1 to maintain backward compatibility.